Bot.DSharp/DSharpClient.cs

```diff
@@ -20,6 +20,7 @@ public class DSharpClient : IBotClient
         public DSharpClient(IServiceProvider serviceProvider)
         {
             serviceProvider.Inject(out m_componentService);
+            serviceProvider.Inject(out m_environment);
 
             var environment = serviceProvider.GetService<IEnvironment>();
```

Comment:
We have 3 different environments here.

1) This local one
2) The `m_environment` you added
3) Direct usage of `Environment` which you used to get the new `deployType`

I think they should all use 1 or 2, not 3.

Reply:
Wow, I did not read the existing code closely

Reply:
Changed to just use 1 throughout. My bad.